CHANGELOG.md

```diff
@@ -5,13 +5,14 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
-### Changed
-- Exclude more recent records from satellites based on hashkey or driving key (#42)
 
-## [Unreleased]
+## [0.6.5] - 2023-05-26
```

Comment:
Shouldn't this be a `0.7.0`? Maybe I did not use the correct labels, but on my PR there were slight behavioural changes.

Reply:
Good point! Done :+1: